app/lib/routes/settings.dart

```diff
@@ -119,6 +229,25 @@ class BellSoundControl extends StatelessWidget {
   }
 }
 
+class ShareBikeControl extends StatelessWidget {
+  const ShareBikeControl(this.bike, {super.key});
+
+  final Bike bike;
+
+  @override
+  Widget build(BuildContext context) {
+    return ElevatedButton(
+      onPressed: () {
```

Comment:
I'm not sure how simple this is but if the user has no internet maybe we should disable this button.

Reply:
Good idea 👍

Reply:
I will try to implement this when I have done everything else.

Reply:
The share list displays a message like 'Unable to obtain shares, check your connection.'